app/templates/rounds/new.hbs

```diff
@@ -1,3 +1,14 @@
+{{nearest-courses store=store nearestCourses=nearestCourses}}
+
+{{#if nearestCourses}}
+  You lookin' for one-a these?
+  <ul>
+    {{#each nearestCourse in nearestCourses}}
+      <li class='u-pointer'><a {{action 'updateCourseInput' nearestCourse}}>{{nearestCourse.name}}</a></li>
```

Comment:
perhaps `nearestCourse` in this inner scope is really `nearbyCourse`? Or even just `course`?